src/lib.rs

```diff
@@ -515,12 +515,25 @@ impl Group {
 /// assert_eq!(faces, ["😀", "😃", "😄", "😁", "😆"]);
 /// ```
 #[inline]
-pub fn iter() -> impl Iterator<Item = &'static Emoji> {
+pub fn iter() -> impl Iterator<Item = &'static Emoji> + Clone {
     crate::gen::EMOJIS
         .iter()
         .filter(|emoji| matches!(emoji.skin_tone(), Some(SkinTone::Default) | None))
 }
 
+/// Returns an iterator over all shortcodes.
+///
+/// # Examples
+///
+/// ```
+/// let shortcodes: Vec<_> = emojis::shortcodes().take(3).collect();
+/// assert_eq!(shortcodes, ["algeria", "pouting_face", "waning_crescent_moon"]);
```

Comment:
This test will break every time we update gemoji because the map is not ordered.